website/docs/r/dashboard.html.markdown

```diff
@@ -0,0 +1,180 @@
+                        "x": 0,
+                        "y": 4,
+                        "rowSpan": 4,
+                        "colSpan": 6
+                    },
+                    "metadata": {
+                        "inputs": [
+                            {
+                                "name": "ComponentId",
+                                "value": "/subscriptions/${data.azurerm_subscription.current.subscription_id}/resourceGroups/myRG/providers/microsoft.insights/components/myWebApp"
+                            }
+                        ],
+                        "type": "Extension/AppInsightsExtension/PartType/AppMapGalPt",
+                        "settings": {},
+                        "asset": {
+                            "idInputName": "ComponentId",
+                            "type": "ApplicationInsights"
+                        }
+                    }
+                }              
+            }
+        }
+    },
+    "metadata": {
+        "model": {
+            "timeRange": {
+                "value": {
+                    "relative": {
+                        "duration": 24,
+                        "timeUnit": 1
+                    }
+                },
+                "type": "MsPortalFx.Composition.Configuration.ValueTypes.TimeRange"
+            },
+            "filterLocale": {
+                "value": "en-us"
+            },
+            "filters": {
+                "value": {
+                    "MsPortalFx_TimeRange": {
+                        "model": {
+                            "format": "utc",
+                            "granularity": "auto",
+                            "relative": "24h"
+                        },
+                        "displayCache": {
+                            "name": "UTC Time",
+                            "value": "Past 24 hours"
+                        },
+                        "filteredPartIds": [
+                            "StartboardPart-UnboundPart-ae44fef5-76b8-46b0-86f0-2b3f47bad1c7"
+                        ]
+                    }
+                }
+            }
+        }
+    }
+}
+DASH
+    }
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Shared Dashboard. This should be be 64 chars max, only alphanumeric and hyphens (no spaces). 
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the dashboard.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `dashboard_properties` - (Required) JSON data representing dashboard body. See above for details on how to obtain this from the Portal. 
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The Dashboard ID.
+
+## Import
+
+Dashboards don't currently support import.
```

Comment:
once we've added import support above - could we add an example in here?

Reply:
yes - added